### ISSUE TITLE
Composer deprecation warning

### ISSUE THREAD

**Issue by boboldehampsink** [pierrestoffe/craft-language-redirector#17]
As of composer 1.10, the following deprecation warning shows up:

```
Deprecation Notice: Class pierrestoffe\languageredirector\variables\LanguageSwitcherVariable located in ./vendor/pierrestoffe/craft-language-redirector/src/variables/LanguageSwitcher.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v2.0.
```

**Comment by pierrestoffe** [comment]
Thanks for reporting the warning, I'll look into it asap.

**Comment by pierrestoffe** [comment]
The deprecation warning was addressed in release 1.3.4 https://github.com/pierrestoffe/craft-language-redirector/releases/tag/1.3.4
